app/models/procurement_area.rb

```diff
@@ -0,0 +1,7 @@
+class ProcurementArea < ActiveRecord::Base
+  validates :name, presence: true
+
+  def self.ordered_by_name
```

Comment:
Only as a matter of choice. Scopes are class methods defined via the Active Record API, I usually prefer writing the method myself as a matter of extensibility, since, for example, the Rails guides advise using them over scopes in case of passing arguments. Happy to replace this with a scope if the team prefers to :)